src/content/developers/tutorials/run-light-node-geth/index.md

```diff
@@ -9,6 +9,66 @@ sidebar: true
 published: 2022-03-04
 ---
 
+<b> Please note that Geth light clients can be very slow to find peers. This is because they rely on full-node operators opting-in to be light servers that the light clients can request data from. There are usually only a small number of light servers available.
+
+Also note that after [The Merge]() a Geth node alone will not be sufficient - a consensus client must be connected to it in order to track the head of the chain.
+</b>
```

Comment:
In Markdown we should avoid HTML tags (`<b>`) if we can use Markdown syntax instead :]